src/backer/enum.py

```diff
@@ -0,0 +1,8 @@
+from enum import Enum
+
+
+class TraitCodex(Enum):
+    EstOnly: str = "EO"  # Only allow establishment events
+    DoNotDelegate: str = "DND"  # Dot not allow delegated identifiers
+    NoBackers: str = "NB"  # Do not allow any backers for registry
+    RegistryBackers: str = "RB"  # Do not allow any backers for registry
```

Comment:
These can be imported directly from keripy `eventing.TraitCodex`. Also not sure why we would just call this file enum

Reply:
these can be removed now

Reply:
updated
ref here: dab7a7ccc924fd29f3db5e45ddc13ff74f7358a7